mitosheet/css/toolbar.css

```diff
@@ -50,6 +50,49 @@
   cursor: pointer;
 }
 
+.mito-toolbar-bottom .select-container,
+.mito-toolbar-bottom .mito-input {
+  height: 22px;
+  font-size: 11px;
+  width: -webkit-fill-available;
+  margin-left: 4px;
+}
```

Comment:
Hmm. We talked about this a bit during the toolbar refactor, but I don't love the lack of encapsulation that this imposes on the components. With this, you can't go to the .tsx and .css file for a component and see the CSS that drives it -- this is bad I think.

At the very least, I'd much rather this class be in `Input.css` so you can really understand what drives it. 

Also, does `-webkit-fill-available` work everywhere?

Reply:
Ok I'll move it in there! Also yeah good point I think I need to add a few different lines to cover the different browsers